grakn-graph/src/main/java/ai/grakn/graph/internal/AbstractGraknGraph.java

```diff
@@ -363,10 +363,15 @@ void checkMutationAllowed(){
 
     //----------------------------------------------Concept Functionality-----------------------------------------------
     //------------------------------------ Construction
-    @Nullable
```

Comment:
Ugh for some reason i read it as it can accept nulls. Which it can't. Will correct